### ISSUE TITLE
Add support for `|>` pipe

### ISSUE THREAD

**Issue by rleyvasal** [pwwang/pipda#14]
It would be great  if  **pipda** supported the `|>` pipe which has become common place in base R, Julia,  and others.

**Comment by pwwang** [comment]
I don't know much about `|>`. Does it behave similarly to `%>%`? One of the problems for `|>` is that `python` doesn't support this operator. Wherever we put it, it will be a syntax error.

**Comment by pwwang** [comment]
Here is a table of operators supported by python.

https://docs.python.org/3/library/operator.html#mapping-operators-to-functions

**Comment by rleyvasal** [comment]
Yes,  `|>` does behave similar to `%>%` . Too bad it cannot be used in Python

**In R**

`mtcars |> head()`


**In Julia's base**
`julia> [1:5;] |> x->x.^2 |> sum |> inv`

**In Julia's Queryverse**
`cars |> @filter(_.Origin=="USA" |> save("us_cars.csv")`

**Comment by pwwang** [comment]
`pipda` actually supports switching piping sign: 

```python
from pipda import register_piping

register_piping('|')

@register_verb(...)
def verb(...):
  ...

data | verb(...)
```

It is fixed to `>>`, however, in `datar`.
